src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -576,6 +576,21 @@
     )
     private Set<ClaimsCostFrom> divorceClaimFrom;
 
+    @CCD(
+        label = "Uploaded supporting documents",
+        hint = "Such as marriage/civil union certificate, proof of name changes, certified translations etc.",
+        typeOverride = TextArea,
+        access = {DefaultAccess.class}
+    )
+    private String uploadedDocuments;
```

Comment:
this is interesting, this should either be the set of documents uploaded or a yes or no for confirmation of document uploaded. Thats what im getting form the field name.

Reply:
👍 I've updated this to be:
```java
private List<DivorceDocument> supportingDocumentMetadata;
```

Reply:
would be nice to test it locally(if you can) and see if it works as expected